com.gitlab.cunidev.Workflow.json

```diff
@@ -0,0 +1,28 @@
+{
+    "app-id" : "com.gitlab.cunidev.Workflow",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "workflow",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=~/.config/activitywatch"
```

Comment:
Do you actually need this? Your application should respect $XDG_CONFIG_DIR environment variable which is overwritten by flatpak to point to correct, sandboxed directory.

Reply:
And that folder belongs to an external tool Workflow cannot do without.

Reply:
Does the application display anything in case activitywatch is not installed?

Reply:
Yes, a generic "empty state" recommending to set it up properly if it's not installed/not running so far. Better error handling planned

Reply:
@barthalion

Reply:
I guess that's fine then.